### PR TITLE
Update sql migration extension to 0.1.12

### PR DESCRIPTION
**Pull request by neetusingh-ms**
Link to build https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=141330&view=artifacts&pathAsName=false&type=publishedArtifacts

### CODE REVIEW

**Review by erpett — APPROVED** [review]

File: extensionsGallery.json [hunk 1]
```diff
@@ -4126,7 +4126,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.33.0"
+									"value": ">=1.35.0"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4129]: FYI - it doesn't matter a whole lot but you should be keeping the engine version in your package.json in line with this : https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-migration/package.json#L13

I'd suggest updating it there now - but for this release we can probably just skip that (just be careful in the future that whenever you make changes to the extension that requires an updated ADS version you should be updating this as well)

**Comment by erpett** [comment]
LMK when you are ready for me to get this added to the gallery and I will merge and push

File: extensionsGallery.json (hunk 1 above)
Comment (neetusingh-ms) [reply extensionsGallery.json:4129]: PR for updating package.json https://github.com/microsoft/azuredatastudio/pull/18462.
Thank You @Charles-Gagnon for keeping track of changes.

**Comment by erpett** [comment]
Pushed to Gallery

**Pull request merged**